apps/client/src/components/Link.js

```diff
@@ -14,17 +15,19 @@ const Link = ({
   ...rest
 }) => {
   const { trackEvent } = useMatomo();
-  const { topic } = useContext(Context);
+  const { topic, trackEvents } = useContext(Context);
 
   // The default category is navigate, it can be overwritten with the prop category.
   // The default action is clickExternalLink, it can be overwritten. The name in action is the topic name
   // There should always be a event name.
   const onClick = () => {
-    trackEvent({
-      category,
-      action: `${action} - ${topic.slug || "home"}`,
-      name: eventName,
-    });
+    if (trackEvents && isProduction) {
+      trackEvent({
```

Comment:
I think it would be better if trackEvent handles `trackEvents && isProduction` by itself. It could be a noop in those cases. Maybe we need to wrap `useMatomo`'s `trackEvent`.